src/helpers/PackedPluginEntityLib.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.25;
+
+import {PluginEntity} from "../interfaces/IPluginManager.sol";
+
+library PluginEntityLib {
```

Comment:
I think this is a duplicate library file with the `Packed` name, we should remove it